ouroboros-network-protocols/ouroboros-network-protocols.cabal

```diff
@@ -130,31 +131,38 @@ library testlib
                        Ouroboros.Network.Protocol.BlockFetch.Direct
                        Ouroboros.Network.Protocol.BlockFetch.Examples
                        Ouroboros.Network.Protocol.BlockFetch.Test
+                       Ouroboros.Network.Protocol.BlockFetch.Codecs
```

Comment:
I'd move this to the `Test` module, `Codecs` is a bit confusing, especially since we have `Codec`; or maybe use a different name like `CDDL` since most of them are types used for the `CDDL` spec (and now also for benchmarking).

Reply:
I have addressed this!